python/cli/prelude_cli/views/scm.py

```diff
@@ -355,6 +355,129 @@ def parse_from_partner_advisory(controller, partner, advisory_id):
             partner=Control[partner], advisory_id=advisory_id
         )
 
+@scm.command("list-policy-exceptions")
+@click.pass_obj
+@pretty_print
+def list_policy_exceptions(controller):
+    """List all policy exceptions"""
+    with Spinner(description="Querying Policy exceptions"):
+        return controller.list_policy_exceptions()
+
+@scm.command("update-policy-exceptions")
+@click.argument(
+    "partner",
+    type=click.Choice(
+        [c.name for c in Control if c != Control.INVALID], case_sensitive=False
+    ),
+)
+@click.option("-e", "--expires", help="Expiry Date (YYYY-MM-DD hh:mm:ss ([+-]hh:mm))", default=None, type=str)
+@click.option("-i", "--instance_id", required=True, help="instance ID of the partner")
+@click.option("-p", "--policy_id", required=True, help="ID of the policy to update")
+@click.option("-s", "--settings", required=True, help="Comma separated list of all setting names to be excluded")
+@click.pass_obj
+@pretty_print
+def update_policy_exception(controller, partner, expires=None, instance_id=None, policy_id=None, settings=None ):
```

Comment:
rm the `=None` on all the function params (the option decorator will define or default the default)

Reply:
also have an extra space before your final parenthesis